keras/utils/layer_utils.py

```diff
@@ -334,7 +359,9 @@ def print_layer(layer, nested_level=0, is_nested_last=False):
                '|' * nested_level)
 
   layers = model.layers
-  for layer in layers:
+  for i, layer in enumerate(layers):
+    if (layer_range) and (i < layer_range[0] or i > layer_range[1]):
```

Comment:
the bracket for layer_range is not needed.

Reply:
Done, thanks.